src/main/groovy/edu/oregonstate/mist/api/InfoResource.groovy

```diff
@@ -0,0 +1,26 @@
+package edu.oregonstate.mist.api
+
+import javax.ws.rs.Path
+import javax.ws.rs.GET
+import javax.ws.rs.Produces
+import javax.ws.rs.core.MediaType
+import javax.ws.rs.core.Response
+
+/**
+ * Information resource class.
+ */
+@Path('/')
```

Comment:
note that this is still relative to the `rootPath` defined in `configuration.yaml`